app/lib/database/DatabaseManager.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const { Sequelize } = require('sequelize');
+const models = require('./models');
+const { Log } = require('@aliceo2/web-ui')
+// const repositories = require('repositories');
```

Comment:
Are you planning to use this in the future? If not, please remove the commented out code.

Reply:
Yea, I'm